x-pack/plugins/alerting/server/types.ts

```diff
@@ -280,3 +285,14 @@ export interface InvalidatePendingApiKey {
 export type RuleTypeRegistry = PublicMethodsOf<OrigruleTypeRegistry>;
 
 export type RulesClientApi = PublicMethodsOf<RulesClient>;
+
+export interface RuleMonitoringService {
```

Comment:
nit: maybe `PublicRuleMonitoringService` to avoid confusion with the `RuleMonitoringService` class that is defined inside `rule_monitoring_client.ts`?